shared/src/main/scala/Context.scala

```diff
@@ -35,18 +35,19 @@ class Context private (
     private val parent: Option[Context] = None,
     val globals: Globals = Globals(),
     val testMode: Boolean = false,
-    var onlineOutput: String = ""
+    val useStack: Boolean = false
```

Comment:
Remind me again why we need to use the top stack and not just the parent's stack?

Reply:
Well that's dependent on whether stack lambdas should operate on the global stack or the parent stack

Reply:
If `useStack` is enabled, then it uses the global stack no matter what. Should it be renamed to `useGlobalStack`?

Reply:
As I said, that depends on whether we want stack lambdas to use the global stack or the parent stack. The behaviour is subject to change

Reply:
Ah, ok